docs/framework/20categories/40operation/decentralization.md

```diff
@@ -1,10 +1,13 @@
 ### Decentralization of Operations
 
-[Third-party Attestation Protocols](../20architecture/messaging.md/#third-party-attestation-protocols) rely on sets of independent parties. For example, in a [Proof of Authority](../20architecture/messaging.md/#proof-of-authority) protocol messages are deemed valid if at least ```M``` of ```N``` parties sign messages. When external parties attempt to audit the security of the protocol deployment, they will reason about the security of the protocol based on the values of ```M``` and ```N```. They will expect that at least ```M``` parties would need to be compromised, or choose to maliciously sign a value, for a malicious message to be trusted. If one party controls multiple signers, then the true security of the system is different to what it first appears. For example, if one party controls ```M - 1``` signers, then an attacker would only need to compromise that party, and one of the other independent parties. 
+[Third-party Attestation Protocols](../20architecture/messaging.md/#third-party-attestation-protocols) rely on sets of independent parties. For example, in a [Proof of Authority](../20architecture/messaging.md/#proof-of-authority) protocol messages are deemed valid if at least ```M``` of ```N``` parties sign messages. The [Third-party Attestation Protocols](../20architecture/messaging.md/#third-party-attestation-protocols) section described a set of considerations that should be taken into account when reviewing these protocols. This section highlights the critical importance of ensuring the operation of a protocol delivers on the security guarantees of the protocol design. 
+
+When external parties attempt to audit the security of the protocol deployment, they will reason about the security of the protocol based on the threshold number of signers (```M```) and the total number of signers (```N```). They will expect that at least ```M``` parties would need to be compromised, or choose to maliciously sign a value, for a malicious message to be trusted. If one party controls multiple signers, then the true security of the system is different to what it first appears. For example, if one party controls ```M - 1``` signers, then an attacker would only need to compromise that party, and one of the other independent parties. 
```

Comment:
nit: suggest using a single pair of backticks instead of triple in all inline references to `M` and `N`